cmake/codecheck/rules/include_cstdlib.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python -tt
+
+"""Checks that cstdlib is included for some of its functions that we use
+commonly.
+
+Older Macs that build with libc++ instead of libstc++ need this.
+"""
+
+import re
+
+FUNCTION_REGEX = re.compile(
+    r"""(\s|std::)(abs|ato[fil]|atoll|strto[dfl]|strtol[ld]|strtoul[l]{0,1}|std::[s]{0,1}rand|malloc|getenv)\(""")
+
+
+def does_include_cstdlib(lines, fn):
+    includes_cstdlib = False
+    for lineno, line in enumerate(lines, 1):
+        if 'include <cstdlib>' in line:
```

Comment:
```suggestion
        if line.startswith('#include <cstdlib>'):
```
No commented out includes for you :P